security/src/main/java/com/fincity/security/dao/UserDAO.java

```diff
@@ -409,4 +414,114 @@ public Mono<Set<ULong>> getUserListFromClientIds(Set<ULong> clientList) {
 
 		return Mono.just(users);
 	}
+
+	public Mono<Boolean> checkPasswordEqual(ULong userId, String newPassword) {
```

Comment:
Please check the authentication service to see how to equal the password, it is not direct equals.

Reply:
updated need to test